test/e2e/test/create_picture_form.test.js

```diff
@@ -0,0 +1,52 @@
+import { Selector } from 'testcafe';
+
+import { SERVER_URL } from "../config";
+import { DEMO_USER, GUEST } from "../roles";
+import PagePictureCreate from "../page-objects/PagePictureCreate";
+import { resetDB, resetUploads } from "../../util/testUtils";
+const fs = require('fs-extra');
+const path = require('path');
+
+fixture("create picture form")
+    .page(SERVER_URL)
+    .before(async ctx => {
+        resetDB();
+        resetUploads();
+    })
+    .beforeEach(async t => {
+        await t.useRole(DEMO_USER);
+    })
+    .afterEach(async t => {
+        await t.useRole(GUEST);
+    });
+
+
+test('upload new picture via create form', async t => {
+    let page = await PagePictureCreate(t).navigateGemaeldeHinzufuegenNavbar();
+    await page.setTitle("Ein neues Gemälde");
+    await page.checkTitle("Ein neues Gemälde");
+    await page.setMaterial("Acryl auf Leinwand");
+    await page.checkMaterial("Acryl auf Leinwand");
+    await page.setDescription("Eine Beschreibung des Gemäldes");
+    await page.checkDescription("Eine Beschreibung des Gemäldes");
+
+
+    //create exhibition and check fields
+    let dialog = await page.neueAusstellungAnlegen();
+    await dialog.setCategoryName("Ausstellung");
+    await dialog.checkCategoryName("Ausstellung");
+    await dialog.setCategoryDescription("Eine Beschreibung der Ausstellung");
+    await dialog.checkCategoryDescription("Eine Beschreibung der Ausstellung");
+    await dialog.clickAusstellungAnlegenBtn();
+
+    // create unique upload file
+    const filename = path.resolve(__dirname, `../pictures/${`twitter-${Math.random() * 100}.png`}`);
+    await fs.copy(path.resolve(__dirname, '../pictures/twitter.png'), filename);
+
+    await t.setFilesToUpload("#uploadFile", filename);

```

Comment:
We do not want css-selctors in our test cases. This should be outsourced into the [page objects](http://devexpress.github.io/testcafe/documentation/recipes/using-page-model.html) to ensure that we can use the same function on different places. Just as you do it with description, material and so on.

The sam applies for the next two lines.